src/Resources/config/doctrine_mongodb.php

```diff
@@ -39,6 +39,7 @@
         ->set('sonata.admin.builder.doctrine_mongodb_form', FormContractor::class)
             ->args([
                 (new ReferenceConfigurator('form.factory'))->ignoreOnInvalid(),
+                (new ReferenceConfigurator('form.registry'))->ignoreOnInvalid(),
```

Comment:
```suggestion
                (new ReferenceConfigurator('form.registry')),
```
I think it's mandatory, probably `form.factory` too.

Reply:
nevermind, it could be done in another PR, maybe there are more like this.

Reply:
indeed. I will fix it on 3.x as well.

Reply:
Yeah there are quite a few `ignoreOnInvalid()` :confused: